tools/system_libs.py

```diff
@@ -781,9 +819,6 @@ def do_create():
     libs_to_link.append((shared.Cache.get('libcompiler_rt_wasm.a', lambda: create_wasm_compiler_rt('libcompiler_rt_wasm.a')), False))
     libs_to_link.append((shared.Cache.get('libc_rt_wasm.a', lambda: create_wasm_libc_rt('libc_rt_wasm.a')), False))
 
-  if shared.Settings.UBSAN_RUNTIME == 1:
-    libs_to_link.append((shared.Cache.get('libubsan_minimal_rt_wasm.a', lambda: create_wasm_ubsan_minimal_rt('libubsan_minimal_rt_wasm.a')), False))
-
```

Comment:
How does ubsan work on regular systems?  Presumably you need to add a linker flag otherwise you'll get undefined symbols?   I guess we can use our dependency system to do one better but I think requiring the linker flag is also ok.

Reply:
`clang` automatically adds the ubsan library to the linker command line.

Reply:
But only if you specify the `-fsanitize..` flag no?

Reply:
Yea. The code you commented on was the old code that parses the command line for `-fsanitize`, but @kripken wanted to use the dependency system, and so it's now doing that.